pkg/config/config_template.yaml

```diff
@@ -1,424 +1,781 @@
-# disable_py3_validation: false
-
-# BETA: Encrypted Secrets
-#
-# This feature is in beta and its options or behaviour might break between
-# minor or bugfix releases of the Agent.
-#
-# The agent can call an external command to fetch secrets. The command will be
-# executed maximum once per instance containing an encrypted password.
-# Secrets are cached by the agent, this will avoid executing again the
-# secret_backend_command to fetch an already known secret (useful when combine
-# with Autodiscovery). This feature is still in beta.
-#
-# For more information see: https://github.com/DataDog/datadog-agent/blob/master/docs/agent/secrets.md
-#
-# Path to the script to execute. The script must belong to the same user used
-# to run the agent. Executable right must be given to the agent and no rights
-# for 'group' or 'other'.
-# secret_backend_command: /path/to/command
-#
-# A list of arguments to give to the command at each run (optional)
-# secret_backend_arguments:
-#   - argument1
-#   - argument2
-#
-# The size in bytes of the buffer used to store the command answer (apply to
-# both stdout and stderr)
-# secret_backend_output_max_size: 1024
-#
-# The timeout to execute the command in second
-# secret_backend_timeout: 5
+## @param enable_gohai - boolean - optional - default: true
+## Enable the gohai collection of systems data.
 #
-{{ end -}}
-{{- if .LogsAgent }}
-# Logs agent
+#  enable_gohai: true
+
+## @param server_timeout - integer - optional - default: 15
+## IPC api server timeout in seconds.
 #
-# Enable logs collection, disabled by default
-# logs_enabled: false
+#  server_timeout: 15
+
+## @param procfs_path - string - optional - default: /proc
+## Some environments may have the procfs file system mounted in a miscellaneous
+## location. The procfs_path configuration parameter provides a mechanism to
+## override the standard default location: '/proc' - this setting trickles down to
+## integrations and affect their behavior if they rely on the psutil python package.
 #
-# logs_config:
+#  procfs_path: /proc
+
+## @param disable_py3_validation - boolean - optional - default: false
+## Disable Python3 validation of python checks.
 #
-#   Enable container log collection for all the containers (see ac_exclude to filter out containers)
-#   container_collect_all: false
+#  disable_py3_validation: false
+
+## @param secret_backend_command - string - optional
+## The Agent can call an external command to fetch secrets. The command is executed maximum
+## once per instance containing an encrypted password.
+## Secrets are cached by the Agent, this avoid executing again the secret_backend_command to fetch an
+## already known secret (useful when combine with Autodiscovery). This feature is still in beta.
+##
+## For more information see: https://github.com/DataDog/datadog-agent/blob/master/docs/agent/secrets.md
+##
+## secret_backend_command is the path to the script to execute. The script must belong to the same user used
```

Comment:
Lets remove this line since secrets is now available on Windows. I would replace it by something like `secret_backend_command is the path to the script to execute. The executable must have specific rights that differ on Windows and Linux: see the documentation complete setup instructions.`. WDYT ?

Also  we try to use the word `executable` instead of `script` since we need a valid win32 binary on windows (aka a python script won't work on windows).